apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -1013,8 +1013,11 @@ configured() ->
 
 for_plugins(ActionFun) ->
     case lists:flatmap(fun(I) -> for_plugin(I, ActionFun) end, configured()) of
-        [] -> ok;
-        Errors -> erlang:error(#{function => ActionFun, errors => Errors})
+        [] ->
+            ok;
+        Errors ->
+            ?SLOG(error, #{function => ActionFun, errors => Errors}),
```

Comment:
Maybe we should use `critical` instead of `error`.
The plugin not running properly is a very serious mistake.
Plan is to add a alarm type in `emqx_alarm` in the future.

Reply:
Also: we cant change this for_plugins/1 function, 
If we enable plugins from API failed ,we should crash and return the error to api.
We only catch the error when emqx start.